FreshRSS/FreshRSS
# document username:password@ syntax in proxy_help

Closes #5497

Changes proposed in this pull request:

- adds `username:password@` to the `proxy_help` string, so that users are aware this is possible.

An alternative would be to add explicit fields for these, then just add them server-side with string manipulation, but the solution I propose here seems good enough to me, and is less likely to introduce bugs.

It's not clear to me whether the non-english strings should have `// TODO` comments added back, just let me know if they should.

How to test the feature manually:

1. Navigate to a page with proxy settings (i.e., adding or modifying a feed).
2. Observe the new help string, and set the proxy address accordingly. 
3. Connect to the feed via the proxy, now with a username and password.

Pull request checklist:
